install/01_clusterautoscaler.crd.yaml

```diff
@@ -1,159 +1,171 @@
-            object represents. Servers may infer this from the endpoint the client
-            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
-          type: string
-        metadata:
-          type: object
-        spec:
-          description: Desired state of ClusterAutoscaler resource
-          properties:
-            balanceSimilarNodeGroups:
-              description: 'BalanceSimilarNodeGroups enables/disables the `--balance-similar-node-groups`
-                cluster-autocaler feature. This feature will automatically identify
-                node groups with the same instance type and the same set of labels
-                and try to keep the respective sizes of those node groups balanced.'
-              type: boolean
-            ignoreDaemonsetsUtilization:
-              description: 'Should CA ignore DaemonSet pods when calculating resource
-                utilization for scaling scaleDown'
-              type: boolean
-            skipNodesWithLocalStorage:
-              description: 'If true cluster autoscaler will never delete nodes with pods with local storage, e.g. EmptyDir or HostPath'
-              type: boolean
-            maxPodGracePeriod:
-              description: Gives pods graceful termination time before scaling down
-              format: int32
-              type: integer
-            podPriorityThreshold:
-              description: 'To allow users to schedule "best-effort" pods, which shouldn''t
-                trigger Cluster Autoscaler actions, but only run when there are spare
-                resources available, More info: https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#how-does-cluster-autoscaler-work-with-pod-priority-and-preemption'
-              format: int32
-              type: integer
-            resourceLimits:
-              description: Constraints of autoscaling resources
-              properties:
-                cores:
-                  description: Minimum and maximum number of cores in cluster, in
-                    the format <min>:<max>. Cluster autoscaler will not scale the
-                    cluster beyond these numbers.
-                  properties:
-                    max:
-                      format: int32
-                      type: integer
-                    min:
-                      format: int32
-                      minimum: 0
-                      type: integer
-                  required:
-                  - min
-                  - max
-                  type: object
-                gpus:
-                  description: Minimum and maximum number of different GPUs in cluster,
-                    in the format <gpu_type>:<min>:<max>. Cluster autoscaler will
-                    not scale the cluster beyond these numbers. Can be passed multiple
-                    times.
-                  items:
+  version: v1
```

Comment:
where are all this changes to the CRD spec are coming from?

Reply:
cc @bison PTAL

Reply:
@enxebre This is the result of running crd generation with the more recent version of `controller-tools`. It looks like crd generation hadn't been run for a while on this repo.

Reply:
To make this a lower impact change, we could leave things as is (not run crd generation) and make those changes later in time, since the change is not required for what we need on the IBM work (all we need is the additional annotations)

Reply:
That sounds good. Let's add annotation manually but leave the script to generate in a follow up PR. So this can be back-ported minimizing impact for 4.4

Reply:
Thanks @enxebre, now changed to only add the annotations. I left the script change in there for when crds are generated in the future.